libs/agno/agno/agent/agent.py

```diff
@@ -615,6 +615,7 @@ def _run(
                             content=model_response_chunk.content,
                             thinking=model_response_chunk.thinking,
                             redacted_thinking=model_response_chunk.redacted_thinking,
+                            reasoning_content=self.run_response.reasoning_content,
```

Comment:
Hmm that would put it on lots of events right?

Reply:
oh actually it shouldn't be here.
this is for the case
```py
if (
                        model_response_chunk.content is not None
                        or model_response_chunk.thinking is not None
                        or model_response_chunk.redacted_thinking is not None
                        or model_response_chunk.citations is not None
):
```
we dont need it here right @dirkbrnd